pgpool2-4.6.yaml

```diff
@@ -0,0 +1,255 @@
+package:
+  name: pgpool2-4.6
+  version: 4.6.0
+  epoch: 0
+  description: Middleware that works between PostgreSQL servers and a PostgreSQL database client.
+  copyright:
+    - license: BSD-3-Clause AND MIT
+  dependencies:
+    provides:
+      - pgpool2=${{package.full-version}}
+    runtime:
+      - merged-usrsbin
+      - wolfi-baselayout
```

Comment:
I think we can omit this given we're already usrmerged.

Reply:
Noted, I added these packages since they were recently merged into older `pgpool2` melange configs.

I've updated changes to omit them.